RDKShell/RDKShell.cpp

```diff
@@ -3676,7 +3676,7 @@ namespace WPEFramework {
 #endif
                 }
 
-                if (type == "Cobalt")
+                if (type == "Cobalt" || type == "YouTube" || type == "YouTubeTV" || type == "YouTubeKids")
```

Comment:
I don't think we provide these runtime types:

`type == "YouTube" || type == "YouTubeTV" || type == "YouTubeKids"`

if you use this these as callsigns, I think it makes sense to make necessary changes in the callsign configuration itself. this code is meant to handle runtime "type", not application instances.

Reply:
Closing the commit as issue was fixed with https://github.com/rdkcentral/rdkservices/pull/4312